plugins/Fable.React.Plugins.fs

```diff
@@ -98,7 +101,7 @@ type ReactComponentAttribute(exportDefault: bool) =
             AstUtil.makeCall (AstUtil.makeImport "createElement" "react") [callee; propsObj]
         | _ -> expr
 
-    override _.Transform(logger, decl) =
+    override _.Transform(logger, _file, decl) =
```

Comment:
I added an extra argument with a handle to the whole file Fable AST in case it's needed later for https://github.com/fable-compiler/Fable/issues/2286 or similar issues.